diff --git a/node_modules/@conform-to/dom/parse.d.ts b/node_modules/@conform-to/dom/parse.d.ts
index 69c4b47..7a20457 100644
--- a/node_modules/@conform-to/dom/parse.d.ts
+++ b/node_modules/@conform-to/dom/parse.d.ts
@@ -1,6 +1,6 @@
 export type Submission<Schema = any> = {
     intent: string;
-    payload: Record<string, any>;
+    payload: Record<string, unknown>;
     error: Record<string, string[]>;
     value?: Schema | null;
 };
